rcos_io/static/js/attendance.js

```diff
@@ -0,0 +1,49 @@
+let user_id_field = document.getElementById("unverified_user");
+let verification_status = document.getElementById("verification_status");
+
+/**
+ * Given a room code and meeting ID, close a room for attendance. 
+ * @param {string} code 
+ * @param {string} meeting_id 
+ */
+let onClose = (code, meeting_id) => {
```

Comment:
Do you feel this client-side approach is better than using a form and normal view on the server?

Reply:
Since there are no data fields, I feel as though using a form is clunky and necessary; really all we need is to send back the data what room we're trying to close. Like we *could* use one, but I don't see too much of an advantage.

Reply:
Hmm, a whole new JS file just for sending requests to 2 routes seems the clunkier option to me, compared to 
```html
<form action="{{ url_for('meetings.close', meeting_id=meeting['id']) }} method="POST">
  <input type="hidden" name="code" value="{{ code }}" />
  <button class="btn btn-danger" type="submit">Close</button>
</form>
```

Reply:
I just imagine someone unfamiliar with the codebase confused trying to find where the calls to `meetings.close` are, and having to jump from Python to Javascript.

Reply:
The extra file is necessary since it will open a new room / code on refresh. This is mainly an issue for verification, since we don't want to generate a new room just to verify someone. I will agree that your solution for replacing the "onClose" is a lot more concise though, I can change that.

Reply:
Resolved by latest changes.